server.py

```diff
@@ -292,80 +303,82 @@ def get_lang_match_score(c: Coach, languages: Dict[str, int]) -> int:
 
 
 @app.route("/")
-async def index(request: sanic.request.Request) -> sanic.response.HTTPResponse:
-    return await sanic.response.file("build/index.html")
+def index() -> flask.Response:
+    return app.send_static_file("index.html")
 
 
-@app.route("/api/v1/coaches", methods=["GET", "POST", "OPTIONS"])
-@app.route("/api/v1/coaches/<coach_id:int>", methods=["GET", "POST", "DELETE"])
-async def api_coaches(
-    request: sanic.request.Request, coach_id: Optional[int] = None
-) -> sanic.response.HTTPResponse:
+@app.route("/api/v1/coaches", methods=["GET", "POST"])
+def api_coaches() -> FlaskReturn:
     """HTTP API for 'coaches'."""
-    headers = COMMON_HEADERS
+    headers = COMMON_HEADERS.copy()
+
+    try:
+        if flask.request.method == "GET":
+            coaches = get_coaches()
+            response = flask.jsonify([c.to_json() for c in coaches])
+        elif flask.request.method == "POST":
+            coach = create_coach(flask.request.json)
+            response = flask.jsonify(coach.to_json())
+        else:
+            assert False, "Unsupported method"
+    except Exception:
+        logger.exception("Unexpected exception on %s route", flask.request.path)
```

Comment:
Have you deliberately chosen to only log in the error case? Seems useful to log all requests coming in, but possible that the logs could become too verbose if requests are frequent.

Reply:
At the moment, this try/except is very general - I think we should also at least log the traceback/exception type here

Reply:
(Same comment for all routes!)

Reply:
Will add some debug logs (this wasn't changed in the migration)

Reply:
Actually not convinced this is needed, presume the below is coming from the flask app logger:
```
127.0.0.1 - - [07/Dec/2020 13:09:55] "GET /api/v1/coaches HTTP/1.1" 200 -